modules/representation_management/app/controllers/representation_management/v0/pdf_generator_base_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  module V0
+    class PdfGeneratorBaseController < ApplicationController
+      service_tag 'lighthouse-veteran' # Is this the correct service tag?
+      before_action :feature_enabled
+
+      skip_before_action :authenticate
+
+      def create
+        render json: {}, status: :unprocessable_entity
+      end
+
+      private
+
+      def form_params
```

Comment:
The params are not flat like this. Some are nested. Here's the link to the ticket that has the json schema we landed on for the 2122 and 2122a PDF requests - https://app.zenhub.com/workspaces/accredited-representation-management-team-64d0dc51d3e8f4788ac6ef96/issues/gh/department-of-veterans-affairs/va.gov-team/85485

The arg to `#params.permit` should look something like this (of course you can break this out into the different methods like you're doing now):

```
veteran: [
  { name: [:first, :middle, :last] },
  :ssn,
  :vaFileNumber,
  :dateOfBirth,
  :serviceNumber,
  { insuranceNumbers: [] },
  { address: [:addressLine1, :addressLine2, :city, :stateCode, :country, :zipCode, :zipCodeSuffix] },
  :phone,
  :email
],
claimant: [
  { name: [:first, :middle, :last] },
  :dateOfBirth,
  :relationship,
  :claimantId,
  { address: [:addressLine1, :addressLine2, :city, :stateCode, :country, :zipCode, :zipCodeSuffix] },
  :phone,
  :email
],
:organizationName,
:recordConsent,
{ consentLimits: [] },
:consentAddressChange
```